### PR TITLE
PCHR-863: Show all task data in Edit Task modal

### PR DESCRIPTION
**Pull request by AkA84**
#### Problem
When the "Edit Task" modal was open, most of the data of the task wasn't displayed on the modal

before

#### Solution
The issue is solved in two steps:
1. The `ng-show` logic wasn't implemented correctly, that is the fields weren't shown if the data was already there
2. For the lookup lists loaded via ajax (assignments, assignees, target), the list given to the lookup directive must come pre-loaded with the item already selected for the task

Additionally, the layout of the static text for the task name has been fixed, as originally it looked a bit too squashed between the columns

after

### CODE REVIEW

File: uk.co.compucorp.civicrm.tasksassignments/views/modal/task.html [hunk 1]
```diff
@@ -74,14 +75,14 @@
             </div>
             <!-- Subject -->
             <div class="row">
-                <div class="col-xs-12 col-sm-6" ng-show="!showFieldSubject">
+                <div class="col-xs-12 col-sm-6" ng-show="!task.subject && !showFieldSubject">
                     <span class="form-control-static">
                         <a href class="btn btn-indent" ng-click="showFieldSubject = true;">
```
Comment (omar-compucorp) [inline uk.co.compucorp.civicrm.tasksassignments/views/modal/task.html]: i don't know but i think it is better for href to have a value .. something like href="javascript:void(0);"

Comment (AkA84) [inline uk.co.compucorp.civicrm.tasksassignments/views/modal/task.html]: That's AngularJS preferred syntax, you don't give `href` a value and it knows that it's a link that shouldn't do anything if clicked

**Comment by omar-compucorp** [comment]
while i can't fully review it since i am not that familier with angular , but i think it looks good but don't forget to squash the commits .

:)

**Comment by danilo-compucorp** [comment]
Looks good. PR approved.

**Pull request merged**